src/resources/js/admin/tickets-commerce-settings.js

```diff
@@ -250,6 +251,53 @@ tribe.tickets.admin.commerceSettings = {};
 			obj.disconnectButton.on( 'click', obj.handleDisconnectClick );
 		}
 
+		$document.on( 'click', obj.selectors.webHookSetUpLink, function( evt ) {
+			evt.preventDefault();
+			const target = $( evt.target );
+
+			const removeIfExists = $document.find( '#tec-tickets-webhook-creation-update' );
+
+			if ( removeIfExists.length ) {
+				removeIfExists.remove();
+			}
```

Comment:
Just use 
```suggestion
			$(document).find( '#tec-tickets-webhook-creation-update' ).remove();
```

Reply:
Im pretty sure this throws a JS error when the element does not exists. I will check it and confirm.

Reply:
Well... it didnt throw an error! Updated